web/packages/teleport/index.html

```diff
@@ -10,7 +10,7 @@
     <meta name="robots" content="noindex" />
     <title></title>
   </head>
-  <body>
+  <body style="overflow: hidden">
```

Comment:
This is a pretty global setting. Are we confident we didn't break the UI in any other places?